modules/tracking/test/test_trackers.cpp

```diff
@@ -571,8 +571,9 @@ TEST(GOTURN, memory_usage)
 {
   cv::Rect2d roi(145, 70, 85, 85);
   cv::Mat frame;
-  cv::Ptr<cv::Tracker> tracker = cv::TrackerGOTURN::create();
-  string folder = cvtest::TS::ptr()->get_data_path() + "tracking/david/data/david.webm";
+  string path_to_extra = cvtest::TS::ptr()->get_data_path();
+  cv::Ptr<cv::Tracker> tracker = cv::TrackerGOTURN::create(path_to_extra + "../dnn/gsoc2016-goturn/goturn.prototxt", path_to_extra + "../dnn/gsoc2016-goturn/goturn.caffemodel");
```

Comment:
Please use `findDataFile`

Reply:
Why? ```get_data_path``` is working properly with ```export OPENCV_TEST_DATA_PATH=/home/user/opencv_extra/testdata```.

Reply:
1. get_data_path is deprecated.
2. it doesn't work with DNN's OPENCV_DNN_TEST_DATA_PATH

Reply:
So, should I then replace ```get_data_path``` in all tests in ```test_trackers.cpp```? It is used 4 more times.

Reply:
Just do not use it in your patch.

Reply:
You have a error message on CI:
```
[ RUN      ] GOTURN.memory_usage
unknown file: Failure
C++ exception with description "OpenCV(3.4.11-dev) /build/precommit-contrib_linux64/3.4/opencv/modules/dnn/src/caffe/caffe_io.cpp:1133: error: (-2:Unspecified error) FAILED: fs.is_open(). Can't open "/build/precommit-contrib_linux64/3.4/opencv_extra/testdata/cv/../dnn/gsoc2016-goturn/goturn.caffemodel" in function 'ReadProtoFromBinaryFile'
" thrown in the test body.
[  FAILED  ] GOTURN.memory_usage (9 ms)
```

@dkurt 's comment guides you how to solve that in the right way to pass CI.

Reply:
This error may appear due to absent or incorrect environment variable ```OPENCV_TEST_DATA_PATH```. Both cases (```findDataFile``` and ```get_data_path```) are properly working with ```export OPENCV_TEST_DATA_PATH=/home/ilyaelizarov/opencv_extra/testdata```.

Reply:
Take a look on `initDNNTests()` implementation which initializes searching paths for DNN models.
Need to migrate this logic here.

On CI model files are stored on share (not in opencv_extra/testdata)